### PR TITLE
Documentation: Add more docs on Animations

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
An overview of the animations package.

## Summary
For helping onboarding and just documenting general context.

## Relevant Technical Choices
NA

## To-do
NA

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [ ] I have added a matching `Type: XYZ` label to the PR

---

### CODE REVIEW

File: docs/animations.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Our serializable animation objects currently have this structure:
+
+```js
+// sample animation object
+{
+  id, // unique identifier for this animation
+  type, // type of animation, `blink`, `bounce` etc.
+  targets, // what elements in the story this animation is targeting
+  ...args // whatever args this type of animation takes ie `ease` | `duration`
```
Comment (sblinde) [inline docs/animations.md]: I think the linter's struggling w/ the `...args` here

**Review by sblinde — DISMISSED** [review]
🐝

Many thanks for this well-written documentation!

File: docs/animations.md (hunk 1 above)
Comment (maxyinger) [reply docs/animations.md]: same. updated it. hopefully fine now

**Review by mwritter — APPROVED** [review]
👍

**Pull request merged**